BismarkUnchained/scenes/Main/Main.gd

```diff
@@ -1,4 +1,18 @@
 extends Node2D
 
-# for any script: get_node("/root/Main")->time_delta
 var time_delta = 1
+
+func get_time_delta():
+	return time_delta
+
+func set_time_delta(new_delta):
+	#Need a buffer on new_delta to account for floating point precision errors
```

Comment:
?

Reply:
I had a really weird bug where when I went at max velocity, it would crash here, but printing out the new time_delta it was showing as 1. I think it was because of floating point precision error, but I'm not 100% sure